cucumber/api/features/secrets.feature

```diff
@@ -27,6 +35,14 @@ Feature: Adding and fetching secrets
     When I GET "/secrets/cucumber/variable/non-existent"
     Then the HTTP response status code is 404
 
+  Scenario: Update a secret for a nonexistent resource
+
```

Comment:
please remove this empty line

Reply:
I also prefer without the empty line however I prefer to keep the style of the file the same for all scenarios and we can create a separate PR to fix the tests style.

Reply:
i see, thanks